src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -28,6 +29,13 @@ public FindCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
+        if (trimmedArgs.contains("n/") && trimmedArgs.contains("c/")) {
+            String[] nameKeywords = parseNameClassId(trimmedArgs);
+            String[] classIdKeywords = parseClassId(trimmedArgs);
+            return new FindCommand(new NameAndClassIdContainsKeywordsPredicate(Arrays.asList(nameKeywords),
+                    Arrays.asList(classIdKeywords)));
+        }
+
         if (trimmedArgs.contains("n/")) {
             String[] nameKeywords = parseName(trimmedArgs);
             return new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
```

Comment:
Shouldn't all of these be abstracted out of FindCommandParser? Referring to AddCommandParser

Reply:
I think this is fine for the if statements and the block